consensus/src/epoch_manager.rs

```diff
@@ -387,7 +387,7 @@ impl EpochManager {
         info!(
             epoch = epoch_state.epoch,
             validators = epoch_state.verifier.to_string(),
-            root_block = recovery_data.root_block(),
+            root_block = %recovery_data.root_block(),
```

Comment:
This does display formatting? :O TIL!

Reply:
magic :) https://github.com/aptos-labs/aptos-core/blob/b1601d23fb157fee4c4932cd093d3809be8094a4/crates/aptos-logger/src/macros.rs#L125